dht/i_partitioner.hh

```diff
@@ -847,8 +848,7 @@ namespace std {
 template<>
 struct hash<dht::token> {
     size_t operator()(const dht::token& t) const {
-        const auto& b = t._data;
-        return read_le<int64_t>(reinterpret_cast<const char*>(b.data()));
+        return bswap_64(t._data);
     }
```

Comment:
Please add a comment here because it now looks odd. It's not the code's fault (it inherited the strangeness), but it should make it less puzzling to readers.

Reply:
done.